scripts/check_menhir_conflicts.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
```

Comment:
It's good practice in bash to first do:
set -e
to stop the script at the first error.
See semgrep/install-scripts/ for script examples.